projects/elife.yaml

```diff
@@ -126,6 +126,17 @@ defaults:
         #       deletion-policy: delete|retain
         #       encryption: false
         s3: {}
+        eks:
+            # eks defaults only used if a 'eks' section present in project
+            #https://docs.aws.amazon.com/eks/latest/userguide/platform-versions.html
+            version: '1.11'
+            subnet-id: subnet-0c60bc3cb24a2816b
+            redundant-subnet-id: subnet-009a04cbefc8e3661
+            worker:
+                type: t2.small
+                desired-capacity: 1
+                min-size: 1
+                max-size: 3
```

Comment:
it's not necessary, it might even be self-evident if one is working with `eks`, but a simple comment might remove some ambiguity as to what `desired-capacity` is referring to. Is it the number of `t2.small` instances in play or the number of containers hosted on each instance?